Packages/com.unity.inputsystem/InputSystem/Editor/AssetImporter/InputActionImporter.cs

```diff
@@ -413,6 +463,36 @@ private static void CheckAndRenameJsonNameIfDifferent(string assetPath)
                 }
             }
         }
+
+        private class InputActionsMigrationPostprocessor : AssetPostprocessor
+        {
+            static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromAssetPaths)
+            {
+                foreach (var assetPath in importedAssets)
+                {
+                    if (!assetPath.EndsWith("." + InputActionAsset.Extension,
+                        StringComparison.OrdinalIgnoreCase))
+                        continue;
+
+                    var diskPath = EditorHelpers.GetPhysicalPath(assetPath);
+                    string original;
+                    try
+                    {
+                        original = File.ReadAllText(diskPath);
+                    }
+                    catch
+                    {
+                        continue;
```

Comment:
What is the intention for suppressing failure to read the file in the migration step? IMO a console warning that migration could not be done would make sense?